roles/container-engine/containerd/tasks/main.yml

```diff
@@ -52,7 +51,7 @@
     - ansible_proc_cmdline['systemd.unified_cgroup_hierarchy'] is not defined or ansible_proc_cmdline['systemd.unified_cgroup_hierarchy'] != '0'
 
 - include_tasks: containerd_repo.yml
-  when: not is_ostree
+  when: not (is_ostree or (ansible_distribution == "Flatcar Container Linux by Kinvolk"))
```

Comment:
Just a question: Does this mean Flatcar Container Linux contains ostree like Fedora CoreOS?

Reply:
No, Flatcar Container Linux doesn't use ostree. But containerd is already installed by default and it doesn't use package manager.